modules/core/src/math/transformation/vpRotationMatrix.cpp

```diff
@@ -924,6 +932,32 @@ vpRotationMatrix vpRotationMatrix::mean(const std::vector<vpRotationMatrix> &vec
   return R;
 }
 
+/*!
+  Perform rotation matrix orthogonalization.
+ */
+void vpRotationMatrix::orthogonalize()
```

Comment:
@chaumett

Feel free to review and comment if this approach is correct or not.

Reply:
Very nice! 
From French, I would have used the term "orthonormalize" instead of "orthogonalize" but this is indeed the correct term in english...